packages/dropdown/src/js/index.ts

```diff
@@ -149,7 +149,7 @@ export const useDropdown = (
       evt.type === 'click' ||
       (evt.type === 'keydown' &&
         'key' in evt &&
-        (evt.key === 'Enter' || 'ArrowDown' || 'ArrowUp'))
+        ['Enter', 'ArrowDown', 'ArrowUp', ' '].includes(evt.key))
```

Comment:
```suggestion
        (evt.key === 'Enter' || evt.key === 'ArrowDown' || evt.key === 'ArrowUp'))
```

This would make the old `||` combination work.  I like the array approach too.

https://caniuse.com/array-includes doesn't work in IE. Could use `indexOf` instead.